modules/claims_api/lib/bgs_service/local_bgs.rb

```diff
@@ -171,7 +171,11 @@ def make_request(endpoint:, action:, body:, key: nil, namespaces: {}, transform_
                               detail: "local BGS Faraday Timeout: #{e.message}")
         raise ::Common::Exceptions::BadGateway
       end
-      soap_error_handler.handle_errors(response) if response.status != 200
+
+      if response.status != 200
+        errors = soap_error_handler.handle_errors(response)
+        return errors
+      end
```

Comment:
Return errors as the handler provides them since parsing below is only constructed for expected success shapes